d={"Name":"John" ,"Surname":"Smith"}

#This throws error, fix it
#print(d["Smith"]

#Solution
print(d["Surname"])

#A KeyError always means Python could not find a key with the name shown next to KeyError (e.g. Smith ).